### PR TITLE
Syncs ECS code from the warehouse

### PR DESCRIPTION
**Pull request by yakloinsteak**

### CODE REVIEW

**Comment by yakloinsteak** [comment]
@eanders When this all passes, I'd like to deploy to qa and then to all of staging. That okay?

**Comment by eanders** [comment]
> @eanders When this all passes, I'd like to deploy to qa and then to all of staging. That okay?

Yes!

**Comment by yakloinsteak** [comment]
@eanders Okay to merge this? I'd like this to be mainstreamed soon.

File: config/deploy/docker/assets/deploy_tasks.open-path-warehouse.sh [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 set -eo pipefail
 
+echo Storing Themed Maintenance Page
+bundle exec rake maintenance:create
+
```
Comment (eanders) [inline config/deploy/docker/assets/deploy_tasks.open-path-warehouse.sh:7]: I don't think we're setup to do this in CAS

**Pull request merged**